spec/amazon_ses_mailer/message_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+RSpec.describe AmazonSesMailer::Message do
```

Comment:
Tests missing for these methods:
- initialize
- deliver
- build_message
- delivering?

Reply:
The provided tests are useless.